posts/2021-01-14-free-nodejs-hosting.md

```diff
@@ -98,87 +102,72 @@ It also has comprehensive [docs](https://render.com/docs?utm_source=geshan.com.n
 
 <img class="center" loading="lazy" src="/images/free-nodejs-hosting/05render-app-command.jpg" title="Configure build and start command for Render web service" alt="Configure build and start command for Render web service">
 
-* It will take a some minutes to build and deploy, you will see an output like the below:
+- It will take a some minutes to build and deploy, you will see an output like the below:
 
 <img class="center" loading="lazy" src="/images/free-nodejs-hosting/06render-app-deployed.jpg" title="Render web service deployment successful" alt="Render web service deployment successful">
 
-* Your application's URL will be visible under the name of the app, click it to check if your app is running. If you click the app while it is being deployed the first time, it will come back with a `404` response. It will take some time to come alive. Then if you hit `<YOUR-APP-URL>/quotes` you will see something like:
+- Your application's URL will be visible under the name of the app, click it to check if your app is running. If you click the app while it is being deployed the first time, it will come back with a `404` response. It will take some time to come alive. Then if you hit `<YOUR-APP-URL>/quotes` you will see something like:
 
 <img class="center" loading="lazy" src="/images/free-nodejs-hosting/07render-app-running.jpg" title="Render web service Up and running successfully" alt="Render web service Up and running successfully">
 
 That was easy to get your Node.js app up and running pretty fast. Next, let’s look at how to deploy the same application on Vercel.
 
-### Vercel
-
-[Vercel](https://vercel.com) (previously known as Zeit) is a great place to deploy frontend apps. Being a layer on top of serverless functions it can also run some other languages and [Node.js is included](https://vercel.com/docs/runtimes#official-runtimes/node-js) in that list. Amongst other good features, a very handy feature with Vercel is you get a unique URL for each pull request which makes testing that particular branch a breeze. They call this deploy preview.
-
-> [Vercel Pricing](https://vercel.com/pricing) is pretty straight forward. Personal projects run free and it integrates with GitHub, BitBucket, and GitLab.
-
-Let's look at how we can deploy our Quotes API Node.js app on Vercel.
-
-#### Deploy Node.js Quotes API on Vercel
+### Genezio
 
-After you have [registered](https://vercel.com/signup) with Vercel and are logged into Vercel, do the following steps:
+[Genezio](https://genezio.com/deployment-platform/) is a great place to deploy both frontend and backend apps. Being a layer on top of serverless functions it can also run some other languages and [Node.js is included](https://vercel.com/docs/runtimes#official-runtimes/node-js) in that list. Amongst other good features, a very handy feature with Genezio is you get a unique URL for each pull request which makes testing that particular branch a breeze. They call this deploy preview.
```

Comment:
This link is still going to vercel @yashcrest  - https://vercel.com/docs/runtimes#official-runtimes/node-js

Reply:
Are you sure of this `with Genezio is you get a unique URL for each pull request which makes testing that particular branch a breeze.` vercel gives it not sure if Genzio gives it too.